### ISSUE TITLE
Works on Greasemonkey

### ISSUE THREAD

**Issue by Carcosian-TMG** [Opinion/slowroads-handling-editor#1]
- The script works using Violentmonkey version 2.13.3. Have yet to try it using Greasemonkey.
- The script _does not_ work on Chrome, and breaks the loading of the original script, which, as far as I can tell, can only be fixed by clearing the cookies for slowroads.io.

Feature Suggestion: Add tooltips to the settings, to make it easier to grasp what each setting affects.

**Comment by Opinion** [comment]
Thanks for the info on Greasemonkey! I'll update the readme later today.

I didn't include any tooltips because I don't actually know how each setting affect the car. Each setting has the same name as the property they are linked to.
Some are more obvious than others, but it seems to me that settings can have multiple effects (even "accel").

**Comment by Opinion** [comment]
I just tested with Greasemonkey but I couldn't get it to work. Greasemonkey handles `window.propertyHere` differently. For Greasemonkey I need to access these properties through `unsafeWindow`.
https://wiki.greasespot.net/UnsafeWindow

To add Greasemonkey support I have to replace all instances of `window`. Make them check `unsafeWindow` first and use `window` as a fallback.

BTW, I also tested the script in Violentmonkey and it works perfectly there without any changes.
